scripts/poststart_hook.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/sh
+
+# Configuration
+PLATFORM="docker"  # Change to "helm" or "cdk" as needed
+VERSION="unknown"
+STATUS=""
+SERVER_HEALTH_URL="http://hyperswitch-server:8080/health"
+HYPERSWITCH_URL="http://hyperswitch-server:8080/health/ready"
```

Comment:
Since we've called the other URL as server health URL, let's call this server deep health check URL:

```suggestion
SERVER_HEALTH_CHECK_URL="http://hyperswitch-server:8080/health"
SERVER_DEEP_HEALTH_CHECK_URL="http://hyperswitch-server:8080/health/ready"
```

Reply:
Can it be :
`HYPERSWITCH_HEALTH_URL`
`HYPERSWITCH_DEEP_HEALTH_URL`
as this was used in helm repo